### ISSUE TITLE
Cursor/target distance measuring works only after clicking in the slice view

### ISSUE THREAD

**Issue by che85** [SlicerProstate/SliceTracker#221]
Cursor/target distance calculation needs to be activated immediately when shortcut key 'd' is pressed without the need to first click into the slice view. 

For now, the click needs to be done, because the slice view doesn't have the focus and that's why the key events don't get processed when the focus is on (let's say) the main SliceTracker UI.

**Comment by fedorov** [comment]
I don't understand the title of the issue, and the description does not help ... :(

**Comment by che85** [comment]
let me rephrase

**Comment by che85** [comment]
Is it better now?

**Comment by fedorov** [comment]
Yes, this is much better. I will make another small refinement :)

**Comment by fedorov** [comment]
Now, about the actual issue - do you think this can actually be solved? I thought it is some Qt limitation. This is the case with many Slicer features.

**Comment by che85** [comment]
Yes it should be possible to solve that. The Segment Editor and the "old" Editor both use the following code:

```
shortcut = qt.QShortcut(slicer.util.mainWindow())
shortcut.setKey( qt.QKeySequence(key) )
shortcut.connect( 'activated()', callback )
self.shortcuts.append(shortcut)
```

but when you use QShortcut, you can only listen to activated signal which means, that it's only active when you press the key, but there is no event for releasing the key. 

I will think about it. I don't think that it is urgent, but would be less confusing to the user if it works properly.

**Comment by fedorov** [comment]
yes, I agree

**Comment by che85** [comment]
Closing this issue since there will be a newer version in the refactored SliceTracker
